src/main/java/com/github/onsdigital/perkin/SurveyListener.java

```diff
@@ -141,7 +149,11 @@ public String test() throws IOException {
 
     @Override
     public void handleRecovery(Recoverable recoverable) {
-        log.warn("QUEUE|CONNECTION|END|queue connection was recovered");
+        log.warn("QUEUE|CONNECTION|END|queue connection was recovered: {}", recoverable);
```

Comment:
it's just an interface to add and remove RecoveryListeners (thought it might have had information about what it was recovering)